### PR TITLE
Update ReadMe Language based on ACP-77 Updates

### PR DESCRIPTION
**Pull request by meaghanfitzgerald**
## Why this should be merged

Updates the explanatory language of the validator manager readme by correcting broken links and transaction type naming that changes with the distinction between _Subnets_ and _L1s_. 

## How this works

- Updated `contracts/validator-manager/README.md` and `contracts/validator-manager/MessageSpec.md` based on most recent version of [ACP-77](https://github.com/avalanche-foundation/ACPs/tree/main/ACPs/77-reinventing-subnets).
- Removes unnecessary capitalization of "validator" and "delegator" except when used to refer to the Validator Manager contract specifically.

### CODE REVIEW

File: contracts/validator-manager/MessageSpec.md [hunk 1]
```diff
@@ -1,185 +1,155 @@
...
+
+Specification:
+
+|          Field |       Type |     Size |
+| -------------: | ---------: | -------: |
+|      `codecID` |   `uint16` |  2 bytes |
+|       `typeID` |   `uint32` |  4 bytes |
+| `validationID` | `[32]byte` | 32 bytes |
+|       `uptime` |   `uint64` |   8 byte |
+|                |            | 46 bytes |
```
Comment (meaghanfitzgerald) [inline contracts/validator-manager/MessageSpec.md:131]: Where is this message spec'd?

File: contracts/validator-manager/MessageSpec.md [hunk 2]
```diff
@@ -1,185 +1,155 @@
...
+### `SubnetToL1ConversionMessage`
 
-- Description: Confirms conversion to a Permissionless Subnet on the P-Chain
+Description: Confirms conversion to L1 on the P-Chain
 
-- Signed by: P-Chain
-
-- Consumed by: Validator Manager contract
+Signed by: P-Chain
+Consumed by: L1
```
Comment (cam-schultz) [inline contracts/validator-manager/MessageSpec.md:8]:
```suggestion
Signed by: P-Chain

Consumed by: L1
```
To add a line break in the formatted doc

File: contracts/validator-manager/MessageSpec.md [hunk 3]
```diff
@@ -1,185 +1,155 @@
...
-### `RegisterSubnetValidatorMessage`
-
-- Description: Registers a Subnet Validator on the P-Chain
-
-- Signed by: Subnet
-
-- Consumed by: P-Chain
-
-Specification:
+The `SubnetToL1ConversionMessage` is specified as an `AddressedCall` with `sourceChainID` set to the P-Chain ID, the `sourceAddress` set to an empty byte array, and a payload of:
```
Comment (cam-schultz) [inline contracts/validator-manager/MessageSpec.md:12]: This line is a bit odd formatting wise. We also don't mention the `AddressedCall` format anywhere else. My preference would be to remove it.

File: contracts/validator-manager/MessageSpec.md
Comment (cam-schultz) [inline contracts/validator-manager/MessageSpec.md:1]: I'd be in favor of removing the specifications for P-Chain Warp messages altogether, and replacing with a link to the [section](https://github.com/avalanche-foundation/ACPs/tree/main/ACPs/77-reinventing-subnets#p-chain-warp-message-payloads) in the ACP.

The only type that we'd need to explicitly specify here is `ValidationUptimeMessage`, as that originates from the subnet.

File: contracts/validator-manager/MessageSpec.md (hunk 1 above)
Comment (cam-schultz) [reply contracts/validator-manager/MessageSpec.md:131]: Defines in subnet-evm here: https://github.com/ava-labs/subnet-evm/blob/75785fac093e02c4bb893f975d59fd3f10e7b298/warp/messages/validator_uptime.go#L14

**Review by cam-schultz — COMMENTED** [review]
We should also update the type definitions in the Solidity contracts

File: contracts/validator-manager/MessageSpec.md
Comment (meaghanfitzgerald) [reply contracts/validator-manager/MessageSpec.md:1]: I agree, I will work that into this PR

**Review by cam-schultz — APPROVED** [review]

File: contracts/validator-manager/README.md [hunk 4]
```diff
@@ -92,49 +96,54 @@ Proof-of-Stake Validator management is provided by the abstract contract `PoSVal
...
 
 ### Convert PoA to PoS
 
-A `PoAValidatorManager` can later be converted to a `PoSValidatorManager` by upgrading the implementation contract pointed to by the proxy. After performing the upgrade, the `PoSValidatorManager` contract should be initialized by calling `initialize` as described above. The Validator set contained in the `PoAValidatorManager` will be tracked by the `PoSValidatorManager` after the upgrade, but will not be eligible to earn staking rewards, nor are they able to be delegated to.
+A `PoAValidatorManager` can later be converted to a `PoSValidatorManager` by upgrading the implementation contract pointed to by the proxy. After performing the upgrade, the `PoSValidatorManager` contract should be initialized by calling `initialize` as described above. The validator set contained in the `PoAValidatorManager` will be tracked by the `PoSValidatorManager` after the upgrade, but will neither be eligible to stake and earn staking rewards, nor provide any such capability for potential delegators.
```
Comment (bernard-avalabs) [inline contracts/validator-manager/README.md:104]:
```suggestion
A `PoAValidatorManager` can later be converted to a `PoSValidatorManager` by upgrading the implementation contract pointed to by the proxy. After performing the upgrade, the `PoSValidatorManager` contract should be initialized by calling `initialize` as described above. The validator set contained in the `PoAValidatorManager` will be tracked by the `PoSValidatorManager` after the upgrade, but these validators will neither be eligible to stake and earn staking rewards, nor support delegation.
```

File: contracts/validator-manager/README.md [hunk 5]
```diff
@@ -92,49 +96,54 @@ Proof-of-Stake Validator management is provided by the abstract contract `PoSVal
...
+
+The `RegisterL1ValidatorMessage` is delivered to the P-Chain as the Warp message payload of a `RegisterL1ValidatorTx`. Please see the transaction [specification](https://github.com/avalanche-foundation/ACPs/tree/main/ACPs/77-reinventing-subnets#registerl1validatortx) for validity requirements. The P-Chain then signs a [`L1ValidatorRegistrationMessage`](https://github.com/avalanche-foundation/ACPs/tree/main/ACPs/77-reinventing-subnets#l1validatorregistrationmessage) Warp message indicating that the specified Validator was successfully registered on the P-Chain.
```
Comment (bernard-avalabs) [inline contracts/validator-manager/README.md:112]:
```suggestion
The `RegisterL1ValidatorMessage` is delivered to the P-Chain as the Warp message payload of a `RegisterL1ValidatorTx`. Please see the transaction [specification](https://github.com/avalanche-foundation/ACPs/tree/main/ACPs/77-reinventing-subnets#registerl1validatortx) for validity requirements. The P-Chain then signs a [`L1ValidatorRegistrationMessage`](https://github.com/avalanche-foundation/ACPs/tree/main/ACPs/77-reinventing-subnets#l1validatorregistrationmessage) Warp message indicating that the specified validator was successfully registered on the P-Chain.
```

**Review by bernard-avalabs — APPROVED** [review]
Just left a few minor suggestions.

**Review by geoff-vball — APPROVED** [review]

**Review by iansuvak — APPROVED** [review]

**Pull request merged**